### DISCUSSION TITLE
When will expo@48 and react-native@0.71 be supported?

### DISCUSSION THREAD

**Discussion by Arnoldnuo** [infinitered/ignite#2397] [Q&A] [answered]

**Comment by frankcalise** [comment, marked as answer]
Hi @Arnoldnuo , I moved this "issue" to Q&A since it's more of a question. You can follow along the PR for the latest Expo and react-native versions here: https://github.com/infinitered/ignite/pull/2377

There are some fixes we're trying to clean up for Expo web first, but feel free to follow along with those changes if you need guidance upgrading
